styles/theme/_uswds-theme-custom-styles.scss

```diff
@@ -523,9 +526,9 @@ main .usa-button--secondary {
   padding-left: 2.75rem;
 }
 
-//Grandchild link
-.usa-sidenav__sublist .usa-sidenav__sublist a {
-  @include u-padding-left(8);
+.usa-sidenav__sublist .usa-sidenav__sublist a,
+.usa-sidenav__sublist a:not(.usa-button) {
+  padding-left: 2.5rem !important;
```

Comment:
? can you tell me more about the !important override here and why you hardcoded the padding value instead of using the USWDS utility mixin?

Reply:
Honestly, I have no idea and we don't have the history with the change lined up with the templates update.  Will revert and we can address when both are merged.